src/status_im2/contexts/wallet/account/tabs/about/view.cljs

```diff
@@ -2,35 +2,53 @@
   (:require
     [quo.core :as quo]
     [react-native.core :as rn]
+    [react-native.platform :as platform]
+    [react-native.share :as share]
     [status-im2.contexts.profile.utils :as profile.utils]
     [status-im2.contexts.wallet.account.tabs.about.style :as style]
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
 (defn about-options
   []
-  [quo/action-drawer
-   [[{:icon                :i/link
-      :accessibility-label :view-on-eth
-      :label               (i18n/label :t/view-on-eth)
-      :right-icon          :i/external}
-     {:icon                :i/link
-      :accessibility-label :view-on-opt
-      :label               (i18n/label :t/view-on-opt)
-      :right-icon          :i/external}
-     {:icon                :i/link
-      :accessibility-label :view-on-arb
-      :label               (i18n/label :t/view-on-arb)
-      :right-icon          :i/external}
-     {:icon                :i/copy
-      :accessibility-label :copy-address
-      :label               (i18n/label :t/copy-address)}
-     {:icon                :i/qr-code
-      :accessibility-label :show-address-qr
-      :label               (i18n/label :t/show-address-qr)}
-     {:icon                :i/share
-      :accessibility-label :share-address
-      :label               (i18n/label :t/share-address)}]]])
+  (let [{:keys [address name]} (rf/sub [:wallet/current-viewing-account])
+        title (str name " address")]
```

Comment:
Generally, It's good to name the variable with usage for better understanding.
We can use `share-title` or `title-for-share`. WDYT?

Additionally, I guess the `address` should be from the translation.

Reply:
if we can rename `name` to `address-name` it'd be good, just to avoid shadowing a core var